radiography/src/main/java/radiography/Views.kt

```diff
@@ -0,0 +1,18 @@
+package radiography
+
+import android.view.View
+
+/**
+ * Extension function for [Radiography.scan] when scanning starts from a specific view.
+ * @see Radiography.scan
+ */
+fun View?.scan(
+  includeTextViewText: Boolean = false,
+  textViewTextMaxLength: Int = Int.MAX_VALUE,
+  viewFilter: ViewFilter = ViewFilter.All
+): String {
+  if (this == null) {
+    return "null"
```

Comment:
Nit: Pull return out of `if`

Reply:
do y'all have a preference on implicit returns vs not?  i lean towards avoiding them since they're not easily readable.

Reply:
In short cases like this I think expression bodies are just as readable, but I don't have a strong opinion either way. For small one-liners I prefer expression bodies.